include/restful_mapper/json.h

```diff
@@ -129,6 +130,20 @@ class Json
     bool is_array() const;
     bool is_map() const;
 
+    bool add_string(const std::string &key, const std::string &string_value, Json::Node &outNode);
+    bool add_int(const std::string &key, int int_value, Json::Node &outNode);
+    bool add_double(const std::string &key, double double_value, Json::Node &outNode);
+    bool add_bool(const std::string &key, bool bool_value, Json::Node &outNode);
+    bool add_array(const std::string &key, Json::Node &outNode);
+    bool add_map(const std::string &key, Json::Node &outNode);
+    
+    bool add_string(const std::string &string_value, Json::Node &outNode);
+    bool add_int(int int_value, Json::Node &outNode);
```

Comment:
Don't we want to use standard types aka: int32_t so we don't run into 64/32 issues.